halpybot/packages/announcer/announcer.py

```diff
@@ -191,9 +180,9 @@ async def announce(self, announcement: str, args: Dict, client: HalpyBOT):
         except AlreadyExistsError as aee:
             logger.exception("Case Already Exists Matching")
             raise AlreadyExistsError from aee
-        except KFCoordsError:
+        except KFCoordsError as kf_err:
             logger.exception("KFCoords were invalid!")
-            raise KFCoordsError
+            raise KFCoordsError from kf_err
```

Comment:
seems odd to capture an exception and then raise an other exception of the same type from it.
Was the intention here to re-raise the exception? if so, there is a simpler syntax for it:

```suggestion
            raise
```